extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/DynamicVerificationKeyResolver.java

```diff
@@ -115,6 +117,12 @@ public Uni<? extends VerificationKeyResolver> apply(JsonWebKeySet jwks) {
                             newKey = jwks.getKeyWithoutKeyIdAndThumbprint("RSA");
                         }
 
+//                        if (newKey == null && tryAll && kid == null && thumbprint == null) {
+//                            LOG.debug("JWK is not available, neither 'kid' nor 'x5t#S256' nor 'x5t' token headers are set,"
+//                                    + " falling back to trying all available keys");
+//                            newKey = jwks.findKeyInAllKeys(jws); // there is nothing to check the signature for in this method
```

Comment:
Can you clarify this comment ?

Reply:
In the code of the `DynamicVerificationKeyResolver` there is no `JsonWebSignature`, but it need such object to perform the brute force algorithm on it with the different keys.
Therefore, I commented out the code, because it's not functional and I didn't see an easy solution to getting this to work there.

Reply:
@c15yi ok, a jose4j instance for verifying a signature can be built manually for this case with some custom basic resolver delegating to your new method,  but I can take care of it later